datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -1316,6 +1327,14 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn read_merged_batches() -> Result<()> {
```

Comment:
I suggest using a enum here (and in the other tests like this below) rather than a bool so it is easier to understand what is happening at the callsite

For example: 
```rust
        _run_read_merged_batches(ForceViews::Yes).await?;
        _run_read_merged_batches(ForceViews::Yes).await?;
```

Reply:
Nice. Done!

Reply:
Thank you -- this is much easier to follow now